server/grpc/server_test.go

```diff
@@ -110,12 +109,11 @@ func (s *IntegrationTestSuite) TestGRPCServer_Reflection() {
 	// Test server reflection
 	ctx, cancel := context.WithTimeout(context.Background(), 15*time.Second)
 	defer cancel()
-	stub := rpb.NewServerReflectionClient(s.conn)
 	// NOTE(fdymylja): we use grpcreflect because it solves imports too
 	// so that we can always assert that given a reflection server it is
 	// possible to fully query all the methods, without having any context
 	// on the proto registry
-	rc := grpcreflect.NewClient(ctx, stub)
+	rc := grpcreflect.NewClientAuto(ctx, s.conn)
```

Comment:
Why not `NewClientV1Alpha` to keep the same behavior?

Reply:
The [deprecation note](https://pkg.go.dev/github.com/jhump/protoreflect@v1.15.0/grpcreflect#NewClient) says:

>  Use NewClientV1Alpha if you are intentionally pinning the v1alpha version of the reflection service. 

I don't imagine we are intentionally pinning v1alpha here, are we?